frontend/src/metabase/meta/memoize.js

```diff
@@ -0,0 +1,19 @@
+
+let memoized = new WeakMap();
```

Comment:
I'm thinking more in terms of when do changes in the metadata get reflected. Would I need to refresh the entire page to kill off the parent instance?